eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -314,8 +314,8 @@
     {% set defaultDraggable = 'true' %}
 
     {% set hasContent = field.value is not null %}
-    {% set latitude = field.value.latitude|default( 0 ) %}
-    {% set longitude = field.value.longitude|default( 0 ) %}
+    {% set latitude = field.value.latitude|number_format( 6, '.', '' )|default( 0 ) %}
```

Comment:
`default` filter should be used before `number_format`. It seems to work like this as well, but that is because `number_format` will cast `null` value, so `default` will never be applied.